admin/app/controllers/authenticated/sessions/session/informations.js

```diff
@@ -49,25 +52,42 @@ export default class IndexController extends Controller {
     } catch (err) {
       this.notifications.error('Erreur lors de la définalisation de la session');
     }
+    this.cancelModal();
+  }
+
+  @action
+  async onUnfinalizeSessionButtonClick() {
+    this.modalMessage = 'Êtes-vous sûr.e de vouloir définaliser cette session ? Cliquez sur confirmer pour poursuivre.';
+    this.modalConfirmAction = this.unfinalizeSession;
+    this.isShowingModal = true;
   }
 
   @action
   async checkForAssignment() {
     if (this.isAssigned) {
-      this.isShowingAssignmentModal = true;
+      this.modalTitle = 'Assignation de la session';
+      this.modalMessage = `L'utilisateur ${this.sessionModel.assignedCertificationOfficer.get(
+        'fullName',
+      )} s'est déjà assigné cette session.<br />Voulez-vous vous quand même vous assigner cette session ?`;
```

Comment:
```suggestion
     this.modalMessage = this.intl.t('tutu', { fullName: this.sessionModel.assignedCertificationOfficer.get('fullName'), htmlSafe: true, });
```

et coté json ça donnerait 
```
"tutu": "L'utilisateur {fullName} s'est déjà assigné cette session.'<br/>'Voulez-vous vous quand même vous assigner cette session ?",
```